### ISSUE TITLE
Allow Sheet To Have Collapse Tree by Default

### ISSUE THREAD

**Issue by Volksters** [crnormand/gurps#1534]
Just a little OCD in me, is it possible to allow the sheet to start as collapse tree instead of expand tree in the inventory, advantages, etc? It could just be a strange occurrence but my players reported that their sheet feels sluggish if the trees are expanded instead of collapse, of course I think that might have to do with the way that data is presented all at the same time.

**Comment by rinickolous** [comment]
Could you elaborate on how the sheet feels sluggish?

**Comment by Volksters** [comment]
The major sluggish feel for us is when my player is trying to add or reduce the amount of items on their sheet.

Say some pieces of copper coin, where the player wants to reduce from 100 to 95. When the player clicks the arrow button to reduce, sometimes we have to wait because the sheet refreshes and then we have to scroll down again if the items are all the way at the bottom.

As for the collapsing part, it's not instantaneous obviously, but it sometimes takes a while for the containers to expand or collapse.

I don't know if this might have to do with us playing on Forge where we're connected onto a server but our latency to the server is relatively good, 15 ms for me so things should be fast.

**Comment by rinickolous** [comment]
Yeah, there's definitely some functions being run unnecessarily in the case of toggling collapsible items. Maybe in the case of item quantity updates too. We'll try to address this soon.
